### PR TITLE
CSA CCM

### PR DESCRIPTION
**Pull request by joedimarzio**
csa_ccm.json is a complete list of all standards for the CCM. I have been able to map all but a few of them to procedures in controls-mapping.json. 

Here are the standards I have been unable to find documented completely through our procedures:
- DSI-06(data ownership/stewardship) -> cp-role-assignment?
- EKM-01(keys must have identifiable owners)
- IVS-02(logging changes to virtual machine image)
- IVS-04(quality/performance of system measured and documented) -> cp-bcdr-app?
- IPY-05(interoperability/portability for virtual machines) -> cp-model-architecture?
- MOS-02(list of application stores)
- MOS-07(application compatibility)
- MOS-09(mobile device inventory)
- MOS-12(jailbreaking)
- MOS-18(remote wipe enabled for devices)
- SEF-01(points of contact for authorities listed)
- STA-05(supply chain agreements)
- STA-07(review of service agreements)
- TVM-03(mobile code) -> cp-access-endpoints?

Based on this list, I think it could be beneficial to add individual procedure documents for 
- interoperability/portability
- virtualization
- mobile devices
- business agreements

### CODE REVIEW

**Comment by erichs** [comment]
Thanks for doing this, @joedimarzio!

File: templates/procedures/cp-model-quality.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### Quality of Service 
+
+{{companyShortName}} strives to provide a high quality of service
+to all of its customers. This is accomplished through a security 
+architecture that encompasses all of {{companyShortName}}'s operations
+and provides high data confidentiality, integrity, and availability.
+
+An overview of {{companyShortName}}'s architecture can be found in 
+cp-model-architecture.md. {{companyShortName}} uses a highly scalable
```
Comment (erkangz) [inline templates/procedures/cp-model-quality.md.tmpl]: let's make these links: [Security Architecture](cp-model-architecture.md)

and later in the doc: [Application Service Event Recovery](cp-bcdr-app.md)

File: templates/procedures/cp-model-quality.md.tmpl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+cloud architecture to provide system quality at all times. 
+
+All systems are monitored and measured in real time as described in 
+cp-bcdr-app.md.
```
Comment (erkangz) [inline templates/procedures/cp-model-quality.md.tmpl]: make this a link

File: templates/procedures/cp-model-quality.md.tmpl [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+and provides high data confidentiality, integrity, and availability.
+
+An overview of {{companyShortName}}'s architecture can be found in 
+cp-model-architecture.md. {{companyShortName}} uses a highly scalable
+cloud architecture to provide system quality at all times. 
+
+All systems are monitored and measured in real time as described in 
+cp-bcdr-app.md. 
+
+{{companyShortName}} uses DevOps methodology as described in cp-sdlc-dev.md
```
Comment (erkangz) [inline templates/procedures/cp-model-quality.md.tmpl]: here too

**Pull request merged**